lua/harpoon/mark.lua

```diff
@@ -144,6 +144,7 @@ M.add_file = function(file_name_or_buf_id)
 
     local found_idx = get_first_empty_slot()
     harpoon.get_mark_config().marks[found_idx] = create_mark(buf_name)
+    M.store_offset()
```

Comment:
THis is the wrong place.  add_file calls create mark which creates a mark with the offset already!

Reply:
You should be able to remove it here, or update create_mark to make sure it puts the rows and cols in.